src/index.html

```diff
@@ -11,17 +11,410 @@
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li
+            data-qa="nav-hover"
+            class="header__item"
+          >
+            <a
+              href="#"
+              class="header__link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="cards">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__image">
+          <img
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+          />
+        </div>
+
+        <div class="card__content">
+          <div class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__reviews">
+            <div class="stars card__stars">
+              <div class="stars__star stars__star--color"></div>
+              <div class="stars__star stars__star--color"></div>
+              <div class="stars__star stars__star--color"></div>
+              <div class="stars__star stars__star--color"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__quantity-reviews">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <span class="card__descr">Price:</span>
+            <span class="card__value">$2,199</span>
+          </div>
+
+          <a
+            href="#"
+            class="card__btn"
+            data-qa="card-hover"
+          >
+            buy
+          </a>
+        </div>
+      </div>
```

Comment:
The code for the product card is repeated multiple times. To follow the DRY (Don't Repeat Yourself) principle, consider creating a reusable component for the product card.